src/js/index.js

```diff
@@ -0,0 +1,99 @@
+const playerFactory = (name, symbol) => {
+  const getName = () => name;
+  const getSymbol = () => symbol;
+  return { getName, getSymbol };
+};
+
+const gameFactory = () => {
+  let boardArray = [['', '', ''], ['', '', ''], ['', '', '']];
+  const player = [];
+  let turn = 0;
+
+  const renderBoard = () => {
+    boardArray.forEach((row, rowIndex) => {
+      row.forEach((cellValue, columnIndex) => {
+        document.querySelector('#board')
+          .querySelector(`.board-row:nth-child(${rowIndex + 1}`)
+          .querySelector(`.board-cell:nth-child(${columnIndex + 1}`)
+          .innerHTML = cellValue;
+      });
+    });
+  };
+
+  const sendMessage = (msg) => {
+    document.getElementById('message').innerHTML = `<p>${msg}</p>`;
+  };
+
+  const isWinner = () => {
+    let finished = false;
+    ['00-01-02', '10-11-12', '20-21-22',
+      '00-10-20', '01-11-21', '02-12-22',
+      '00-11-22', '20-11-02'].forEach((move) => {
+      const line = move.split('-');
+      if (boardArray[line[0][0]][line[0][1]] === boardArray[line[1][0]][line[1][1]]
+        && boardArray[line[1][0]][line[1][1]] === boardArray[line[2][0]][line[2][1]]
+        && boardArray[line[0][0]][line[0][1]] !== '') {
+        finished = true;
+      }
+    });
+    return finished;
+  };
```

Comment:
You can greatly simplify this winning condition, by using am array to create the winning combinations, the check every move by a player, if that players move, is in the array.